Dockerfile

```diff
@@ -7,10 +7,12 @@
 
 FROM python:3.6
 
-RUN pip install pipenv
+RUN pip install poetry
 
 WORKDIR /opt/inspire
 
 COPY . /opt/inspire
 
-RUN pipenv install --skip-lock
+# install dependencies
+RUN poetry install --no-interaction --no-ansi
```

Comment:
what's the ``--no-ansi``?

Reply:
From poetry docs: `--no-ansi: Disable ANSI output.` in short: Output will be less preatty, but it will be more clean in logs.